<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveFrame</class>
 <widget class="QFrame" name="ReceiveFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1270</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px; background-color: #111; color: #aaa;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>1270</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ddd; 
font-size: 20px;
font-family: Poppins;</string>
      </property>
      <property name="text">
       <string>BACKUP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_backButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #aaa; background-color: #333; border-top: 3px solid orange;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>1261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #111;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextBrowser" name="m_trackingKey">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>1101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5e5e5;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; \nfont-size: 16px;\nfont-family: Poppins;</string>
       </property>
       <property name="text">
        <string>TRACKING KEY (VIEW ONLY)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_copyTrackingKeyButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #222; border: none;</string>
       </property>
       <property name="text">
        <string>COPY TO CLIPBOARD</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>1261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #111;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; \nfont-size: 16px;\nfont-family: Poppins;</string>
       </property>
       <property name="text">
        <string>MNEMONIC SEED</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="m_seed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>1211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5e5e5;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Your wallet does not support mnemonic seeds. Please generate a new wallet in order to get the 25 word Mnemonic seed.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_copySeedButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #222; border: none;</string>
       </property>
       <property name="text">
        <string>COPY TO CLIPBOARD</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>1261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #111;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="m_copyGUIKeyButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #222; border: none;</string>
       </property>
       <property name="text">
        <string>COPY TO CLIPBOARD</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; \nfont-size: 16px;\nfont-family: Poppins;</string>
       </property>
       <property name="text">
        <string>GUI KEYS</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="m_guiKey">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>1111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5e5e5;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>1261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #111;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="m_copySpendKeyButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #222; border: none;</string>
       </property>
       <property name="text">
        <string>COPY TO CLIPBOARD</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; \nfont-size: 16px;\nfont-family: Poppins;</string>
       </property>
       <property name="text">
        <string>SECRET SPEND KEY</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>20</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; \nfont-size: 16px;\nfont-family: Poppins;</string>
       </property>
       <property name="text">
        <string>PRIVATE VIEW KEY</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="m_spendKey">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>501</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5e5e5;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="m_viewKey">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>50</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #e5e5e5;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Courier'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_copyViewKeyButton">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>90</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #222; border: none;</string>
       </property>
       <property name="text">
        <string>COPY TO CLIPBOARD</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="m_backupButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #aaa; background-color: #333; border-top: 3px solid orange;</string>
      </property>
      <property name="text">
       <string>BACKUP WALLET</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1095</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copySeedButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>copySeedClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copySpendKeyButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>copySpendKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyTrackingKeyButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>copyTrackingKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyViewKeyButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>copyViewKeyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyGUIKeyButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>copyGUIClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backupButton</sender>
   <signal>clicked()</signal>
   <receiver>ReceiveFrame</receiver>
   <slot>backupClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>985</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>backClicked()</slot>
  <slot>copyGUIClicked()</slot>
  <slot>copySpendKeyClicked()</slot>
  <slot>copyViewKeyClicked()</slot>
  <slot>copyTrackingKeyClicked()</slot>
  <slot>copySeedClicked()</slot>
  <slot>backupClicked()</slot>
 </slots>
</ui>
